site/src/pages/release-notes/1.24.0.mdx

```diff
@@ -0,0 +1,279 @@
+---
+date: 2023-06-15
+---
+
+## 🌟 New features
+
+- **Unix Domain Socket Support**: You can now use [Unix domain socket](https://en.wikipedia.org/wiki/Unix_domain_socket)
+  to serve and send requests which is useful for configuring a service mesh to communicate with a sidecar. #1837 #4846
+  ```java
+  import com.linecorp.armeria.common.util.DomainSocketAddress;
+
+  DomainSocketAddress domainSocketAddress = 
+    DomainSocketAddress.of(Paths.get("/tmp/armeria.sock"));
+
+  Server
+    .builder()
+    .http(domainSocketAddress)
+    ...
+    .build();
+
+  WebClient
+    .builder("http://" + domainSocketAddress.authority())
+    ...
+    .build(); 
+  ```
+- **WebSocket Support**: You can now send and receive data over [WebSocket](https://en.wikipedia.org/wiki/WebSocket)
+  using <type://WebSocketService>. #1076 #3904
+  ```java
+  Server
+    .builder()
+    .service("/chat", WebSocketService.of((ctx, messages) -> {
+        WebSocketWriter webSocketWriter = WebSocket.streaming();
+        // Write frames using back pressure.
+        return webSocketWriter;
+     }));
+  ```
+- **Customizable Logging**: You can now use <type://LogFormatter> and <type://LogWriter> to customize the log
+  format and consumer. #3918 #4267 #4943
+  ```java
+  // Convert RequestLog to JSON.
+  LogFormatter logFormatter = LogFormatter.ofJson();
+  LogWriter logWriter =
+    LogWriter
+      .builder() // Use SLF4J logger for logging.
+      .logFormatter(logFormatter)
+      .logger(logger)
+      .build();
+  
+  LoggingClient
+    .builder()
+    .logWriter(logWriter)
+    .newDecorator();    
+  ```
+- **BlockHound Integrations**: Armeria now provides BlockHound integrations, offering enhanced support for
+  detecting with blocking calls in event loops. #4493
+- **Delayed Request Abortion**: You can now delay aborting the <type://HttpRequest> when the <type://HttpResponse>
+  completes on the client side. This may be useful when you want to send additional data even after the response is complete. #4913
```

Comment:
This applies to both sides. I've added this feature to the server-side in the WebSocket PR.
```suggestion
  completes. This may be useful when you want to send additional data even after the response is complete. #4913
```